bin/mutatex

```diff
@@ -286,9 +286,51 @@ else:
     repaired_pdbs_list = [repair_runs[i].working_directory+"/"+current_version.repair_pdb_output_fname(pdbs_list[i]) for i in range(len(pdbs_list))]
     log.info("list of PDBs to be used: %s" % ", ".join(repaired_pdbs_list))
 
+##Check if the repair step removed any residues
 
-# PHASE TWO: mutate + energy
+#get the original PDB list and residues
+pdbs_list = [ os.path.join(main_dir, pdb) for pdb in pdbs_list ]
+original_residues_dictionary = {}
 
+for pdb in pdbs_list:
+    try:
+        key = os.path.splitext(os.path.basename(pdb))[0]
+        original_residues_dictionary[key] = get_foldx_sequence(pdb, multimers=args.multimers)
+    except IOError:
+        exit(1)
+    except ValueError as e:
+        log.error(f"{e}. Exiting...")
+        exit(1)
+
+#the repaired PDB list and residues
+residues_dictionary = {}
+for pdb in repaired_pdbs_list:
+    try:
+        key = os.path.basename(pdb).split('_Repair')[0]
```

Comment:
this has the same problem as we were considering for the other line - in case there's already "_Repair" in the filename for some reason

maybe we can just (a bit hacky but can't think of anything else):

```
key = os.path.basename(pdb)[:-11]
```